README.md

```diff
@@ -1,40 +1,71 @@
 # React, React-Bootstrap, and ES-6 on Rails via WebPack
 
-By Justin Gordon, http://www.railsonmaui.com
+By Justin Gordon and the Rails On Maui Team, http://www.railsonmaui.com
 
-Full tutorial can be found at: [Fast Rich Client Rails Development With Webpack and the ES6 Transpiler](http://www.railsonmaui.com/blog/2014/10/02/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/)
+- Please email us at [justin@railsonmaui.com](mailto:justin@railsonmaui.com) if you have a ReactJs +
+  Rails project.
+- Please file issues for problems and feature requests.
+- Pull requests are welcome!
+- If this work interests you and you are looking for full or part-time remote work, please
+  [click here](http://forum.railsonmaui.com/t/railsonmaui-is-hiring-and-partnering-part-time-remote-is-ok/156).
+
+A Full tutorial article can be found at: [Fast Rich Client Rails Development With Webpack and the ES6 Transpiler](http://www.railsonmaui.com/blog/2014/10/02/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/)
+Note, this source code repository is going to be ahead of the tutorial. We plan to update the tutorial soon.
 
 [Discussion forum regarding the tutorial](http://forum.railsonmaui.com/t/fast-rich-client-rails-development-with-webpack-and-the-es6-transpiler/82/10)
 
+# Example Application
+This is a simple example of using ReactJs for Comments that demonstrates the front end code in both
```

Comment:
How about?

"This is a simple example application that illustrates the use of ReactJs to implement a commenting system. Front-end code leverages both ReactJs and Rails asset pipeline while the backend is 100% Rails."

Or just rephrase that sentence, it wasn't clear for me...